### DISCUSSION TITLE
how do i make this run at startup witout cmd window in the backgroun?

### DISCUSSION THREAD

**Discussion by Theletha** [dokan-dev/dokany#1023] [Q&A] [answered]
Hey!

I managed to get the command to work but i would like something more permanent. Is it possible to make the command run without a cmd window having to be open in the background?
Also is it possible to have it set a different name when it gets mounted instead of the default drive name since it resets every time its mounted?

This .bat file works if i run it as admin:
```
cd "C:\Program Files\Dokan\Dokan Library-1.5.0\sample\mirror"

mirror.exe -r \\Teletha-Qnap\Main -l E
```

**Comment by Liryna** [comment, marked as answer]
Hi @Theletha ,

Regarding the mount without window, you might be interested to run it as a service like [here](https://github.com/dokan-dev/dokany/discussions/1005)
for the naming, could it be the same request as https://github.com/dokan-dev/dokany/discussions/1022 ? If yes, can you provide a screenshot there.
